spec/system/child_object_spec.rb

```diff
@@ -17,10 +17,7 @@
 
   it "creates child objects" do
     parent_object
-    visit(child_objects_path)
-    expect(page).to have_content("1042003")
-    expect(page).to have_link("Edit")
-    click_on("Edit")
```

Comment:
now that the table on the child objects page is a datatable, this spec and the spec file that was deleted below no longer applied as is.